build.yaml

```diff
@@ -117,3 +117,26 @@ jobs:
                 pathtoPublish: '$(System.DefaultWorkingDirectory)/drop'
                 artifactName: 'drop'
             displayName: publish drop
+
+    - job: 'ubuntu_build'
+
+      pool:
```

Comment:
did u check if there is a way to import yaml file to avoid duplicate code?

Reply:
Nop. It's a good point, a little out of the discussion scope but I'll have a look

Reply:
Done... I think we can take a second look a this later as right now my objective is to make sure we add a new job to run things under ubuntu!